src/app/favlist/page.tsx

```diff
@@ -1,11 +1,46 @@
+import Link from "next/link";
 import { type Metadata } from "next";
+import Button from "~/clients/components/Button";
+import Text from "~/clients/components/Text";
+import Card from "~/clients/components/FavCards";
 
 export const metadata: Metadata = {
-  title: "Favorite"
+  title: "Map",
+  description: "Generated by create-t3-app",
 };
 
-const FavList = () => {
-  return <div>FavList</div>;
+const FavPage = () => {
+  return (
+    <div
+      style={{
+        display: "flex",
```

Comment:
I just would like to make sure that the reason why you use inline style here instead of using Mantine styling is not to make `src/app/favlist/page.tsx` a client component. Is that correct?

Reply:
Sorry I forgot to change the structure of it. 
Could you remind me how would you do it? 
I created a folder 'favCard' under components and have FavCard.tsx and FavCards.tsx. FavCards will take multiple favCard components and in favlist page, I import FavCards  components instead of importing FavCard component. 

Is it something how you said you would? 


![Screenshot 2023-06-30 at 8 01 10 AM](https://github.com/van-squad/map-t3/assets/70562492/69f498d2-8d3e-4f0c-919b-1e10e253d155)

![Screenshot 2023-06-30 at 8 00 36 AM](https://github.com/van-squad/map-t3/assets/70562492/1df97953-9a01-41fb-a03c-a52e936b1c6a)

Reply:
@SachiGoto Yep, the basic concept is correct! And for the sake of creating clear folder structures, I would like to share a suggestion like `src/clients/components/pages/Favlist/FavlistPage/index.tsx`. (Plz refer to the screen shots as well)

It is just an idea but we can create `pages` directory inside `src/clients/components`. Then add `Favlist/FavlistPage/index.tsx` (FavlistPage component which shows UI of favlist page). Finally, in FavlistPage component, we can import `FavCards` component to show the contents of Card components.

<img width="174" alt="スクリーンショット 2023-07-01 9 39 00" src="https://github.com/van-squad/map-t3/assets/65790344/397db7e0-fa67-4994-a004-6b10a17b7fa2">
<img width="724" alt="スクリーンショット 2023-07-01 9 39 19" src="https://github.com/van-squad/map-t3/assets/65790344/f4481357-3b18-447e-b286-b67552bb279e">
<img width="866" alt="スクリーンショット 2023-07-01 9 39 40" src="https://github.com/van-squad/map-t3/assets/65790344/56de0d26-5718-4dd1-a418-e89ba2ab0eca">

Reply:
Okay!  Thank you so much for clarifying! That makes sense. Just one question, why Favlist/ FavlistPage? I felt like just having a FavlistPage under pages makes more sense? Also, is there any document that I can refer to? I'd love to learn conventional way of managing files and folders!

Reply:
@SachiGoto Actually, it is just a preference to set `clients/pages/Favlist/FavlistPage/index.tsx` based on the folder structure in my previous company (a Japanese startup company). `Favlist` stands for the same name as `favlist` in `src/pages/favlist` to make us easy to understand where `Favlist/FavlistPage/index.tsx` is used for. So like you said, it's also ok to set like `clients/pages/FavlistPage/index.tsx`.

We should discuss this topic in the next meeting to ensure that everyone is on the same page about the folder structure.

Reply:
As @keento0809 and @devjoylee mentioned, we can talk about folder structure in the next meeting and we can leave it as is for now.
I just put my opinion here: I think we can leverage the app router's [Safe colocation by default](https://nextjs.org/docs/app/building-your-application/routing/colocation#safe-colocation-by-default) in this case.
We can put a components folder inside of each route folder to discriminate route-specific components and general components: FavList and Button respectively for example. I think this will be simpler and more intuitive to find components in the future. And I think we don't need the client folder for just putting components though I'm the one who created it😂 Just putting directly under /src is fine for me.

```
// current
├── app
│   ├── favlist
│   │   ├── page.tsx
├── components
│   ├── pages
│   │   ├── FavlistPage.tsx

// my suggestion
├── app
│   ├── favlist
│   │   ├── components
│   │   │   ├── Favlist // and what ever we need for favlist page
│   │   │   │   ├── intex.tsx // like @devjoylee mentioned #21 
│   │   │   │   ├── styles.ts 
│   │   ├── page.tsx
├── components
│   ├── Button
│   │   ├── Button.tsx // fyi: this is what I usually do with styled-components 
│   │   ├── Button.styles.tsx

```

Reply:
@Lada496 Woa, this folder structure makes us way easier to follow, I like this sophisticated idea!

Reply:
@keento0809 yeah this is the power of app router I finally see it haha
if it's page router we can't put the components folder under `pages` because Next.js detects it as a nested route but app router doesn't, that's why I guess your company had to move the page-specific components outside.

Reply:
That's great! Making the most out of app router!